istio-telemetry/kiali/templates/ingress.yaml

```diff
@@ -0,0 +1,38 @@
+{{- if .Values.kiali.ingress.enabled -}}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
```

Comment:
I see you added the Gateway in ingress - can we get rid of this ?

Reply:
done